tests/scripts/run_chatbot.py

```diff
@@ -21,30 +25,54 @@
 greeting_node = GreetingNode(model)
 collecting_info_node = CollectingInfoNode(model, logger)
 clarify_request_node = ClarifyRequestNode(model)
+meeting_details_service = MeetingDetailsService(model, logger)
+human_schedule_meeting_more_info_node = HumanScheduleMeetingMoreInfoNode(logger)
+prompt_missing_meeting_details_node = PromptMissingMeetingDetailsNode(logger, meeting_details_service)
 
 class ChatBot:
     def __init__(self, graph):
         self.graph = graph
         self.thread_id = "conversation_1"
         self.config = {"configurable": {"thread_id": self.thread_id}}
-    
+        self.initial_state = MeetingMuseBotState(
+            messages=[],
+            user_intent=None,
+            meeting_details=MeetingFindings()
+        )
+    def get_last_message(self, events: dict):
+      # Skip interrupt events
+        if "__interrupt__" in events:
+            return None
+
+        # Get the state from the node result
+        for node_name, state in events.items():
```

Comment:
`node_name` is not used
```suggestion
        for state in events.values():
```

Reply:
nope, this wont work, events.items() gives a [tuple](https://docs.python.org/3/library/stdtypes.html#tuples), and when we do  node_name, state -> we are unpacking the tuple to node_name and state, 

for reference
```python
abc = {
    "arjun": "one",
    "da": "2"
}

for v in abc.items():
    print(abc)

for k, v in abc.items():
    print(f"{k}: {v}")
```
gives
```python
{'arjun': 'one', 'da': '2'}
{'arjun': 'one', 'da': '2'}
arjun: one
da: 2
```